dariobmibar/src/main/res/layout/dario_bmi_dot_indicator.xml

```diff
@@ -0,0 +1,156 @@
+            android:visibility="invisible"
+            android:src="@drawable/bmi_dashed_divider"
+            app:layout_constraintBottom_toTopOf="@+id/textviewBar0"
+            app:layout_constraintStart_toEndOf="@+id/bar1"
+            app:layout_constraintEnd_toStartOf="@id/bar2"/>
+        <View
+            android:id="@+id/bar2PlaceHolder"
+            android:layout_width="0dp"
+            android:visibility="invisible"
+            android:layout_height="0dp"
+            app:layout_constraintBottom_toTopOf="@+id/textviewBar0"
+            app:layout_constraintStart_toEndOf="@+id/bar2"
+            app:layout_constraintEnd_toStartOf="@id/bar3"/>
+
+        <TextView
+            android:id="@+id/textviewBar0"
+            style="@style/BMITextBar"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="18.5"
+            android:visibility="invisible"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toEndOf="@id/divider0"
+            app:layout_constraintStart_toStartOf="@+id/divider0" />
+
+        <TextView
+            android:id="@+id/textviewBar1"
+            style="@style/BMITextBar"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="25"
+            android:visibility="invisible"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toEndOf="@+id/circleIndicator"
+            app:layout_constraintStart_toStartOf="@+id/circleIndicator" />
+        <TextView
+            android:id="@+id/textviewBar2"
+            android:text="30"
+            android:visibility="invisible"
+            style="@style/BMITextBar"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            app:layout_constraintStart_toStartOf="@+id/bar2PlaceHolder"
+            app:layout_constraintEnd_toEndOf="@id/bar2PlaceHolder"
+            app:layout_constraintBottom_toBottomOf="parent"/>
+
+
+<!--        18.5454545-->
+        <ImageView
+            android:id="@+id/selectionLineIndicator"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:visibility="invisible"
+            app:layout_constraintHorizontal_bias="0.76"
+            android:src="@drawable/bmi_selected_indicator"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintTop_toTopOf="@id/bar1"
+            app:layout_constraintBottom_toBottomOf="@id/bar1"
+            android:layout_marginBottom="34dp" />
+
+        <TextView
+            android:id="@+id/bmiText"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            tools:text="Healthy"
+            android:textSize="14sp"
```

Comment:
wrong size for the text, please look at the figma design.

Reply:
changes done